ui/apos/apps/index.js

```diff
@@ -25,6 +26,46 @@ export default () => {
     window.removeEventListener('beforeunload', warningImport);
   }
 
+  async function handleDifferentLocale(event) {
+    const continueImport = await apos.modal.execute('AposModalConfirm', event);
+
+    if (continueImport) {
+      try {
+        const moduleAction = apos.modules[event.moduleName].action;
+
+        await apos.http.post(`${moduleAction}/import`, {
+          body: {
+            overrideLocale: true,
+            exportPath: event.exportPath
+          }
+        });
+      } catch (error) {
+        apos.notify('aposImportExport:importFailed', {
+          type: 'danger',
+          dismiss: true
+        });
+      }
+
+      return;
```

Comment:
we agree that in this case the export file is cleaned from the import route directly?

Reply:
yes, it should be done at [line 107](https://github.com/apostrophecms/import-export/pull/48/files#diff-ddfb11c4e5be78f1cec7898f8526ac6179abe026e1715283367c6eee461f378aR107), as in a "normal" case where no locale overriding is involved